src/inject/fallback.ts

```diff
@@ -1,11 +1,34 @@
+import {createNodeAsap} from './utils/dom';
+
 if (
     matchMedia('(prefers-color-scheme: dark)').matches &&
     !document.querySelector('.darkreader--fallback')
 ) {
-    const fallbackStyle = document.createElement('style');
-    fallbackStyle.textContent = 'html, body, body :not(iframe) { background-color: #181a1b !important; border-color: #776e62 !important; color: #e8e6e3 !important; }';
-    document.documentElement.appendChild(fallbackStyle);
-    fallbackStyle.classList.add('darkreader');
-    fallbackStyle.classList.add('darkreader--fallback');
-    fallbackStyle.media = 'screen';
+    createOrUpdateFallback('html, body, body :not(iframe) { background-color: #181a1b !important; border-color: #776e62 !important; color: #e8e6e3 !important; }');
+}
+
+export function createOrUpdateFallback(css: string) {
+    createNodeAsap({
+        selectNode: () => document.querySelector('darkreader--fallback'),
+        createNode: (target) => {
+            const fallbackStyle = document.createElement('style');
+            fallbackStyle.textContent = css;
+            target.appendChild(fallbackStyle);
+            fallbackStyle.classList.add('darkreader');
+            fallbackStyle.classList.add('darkreader--fallback');
+            fallbackStyle.media = 'screen';
+        },
+        updateNode: (existing) => {
```

Comment:
I think `updateNode` can be an empty NOOP function in this case, like `Function.prototype`. If there will be a fallback style, it will not do anything.